.build/cspell-words.txt

```diff
@@ -58,6 +58,7 @@ hostnames
 HRESULT
 hsts
 httperr
+ietf
```

Comment:
Can we remove this from the dictionary and do something like "ieTf" in the string and do a `.ToLower()` to avoid adding these odd words into the dictionary list.

Reply:
Isn't that what the words list exists for? `ietf` (in lower case) is commonly used. Doing something like ieTf and then doing a `.ToLower()` to make spellcheck happy is not necessarily the most beautiful solution IMO.

Reply:
Okay, let's keep it and do another commit for Test-HMAEAS for line 259 and make that change there as well. Could either be this PR or a different PR.

Reply:
Done